src/utils.js

```diff
@@ -35,7 +35,7 @@ const utils = {
         }
         parts.push(data.data.issue.project.slug);
         parts.push('|');
-        parts.push(`<a href="${data.data.issue.web_url}">${data.data.issue.title}</a>`);
+        parts.push(`${data.data.issue.title} <a href="${data.data.issue.web_url}">(event)</a>`);
```

Comment:
All fields are escaped and/or safe?
- title contains no HTML
- web_url contains no quote
- event contains no HTML

Reply:
Matrix clients are expected to clean any HTML they output as per [spec](https://spec.matrix.org/unstable/client-server-api/#mroommessage-msgtypes). I'm not entirely sure we should be duplicating the effort when creating messages? :thinking:  I could add a cleaner but it seems a bit of a waste effort considering this proxy is meant to be operated in a safe environment where the organization running it controls the secret key needed to post to this proxy.

Of course Sentry itself might create an unsafe message or someone might trigger an error which gets passed via Sentry via this proxy to a client. But again, Matrix clients should by spec not render dangerous HTML to users.